pgx/src/datum/time_with_timezone.rs

```diff
@@ -12,6 +12,7 @@ use crate::{pg_sys, FromDatum, IntoDatum, PgBox};
 use std::ops::{Deref, DerefMut};
 use time::format_description::FormatItem;
 
+/// A `time with timezone` type from PostgreSQL
```

Comment:
TIL that's an actual type name with Postgres and not the version with underscores.